wordpress-mobile/WordPress-iOS
# [Notifications P1] Hide the bottom tab bar for all notification details screens

Fixes #22474

This PR contains changes that allow all types of notification details to be opened on a full screen with no bottom tab bar for iPhones. The split screen with the bottom bar remains unchanged for iPads.

https://github.com/wordpress-mobile/WordPress-iOS/assets/16563318/1a805d2d-4f59-433d-838c-f1c675cb2121

To test: 
- Run the app on an iPad.
- Go to the Notifications tab.
- [ ] Make sure you see a split screen with a notification list on the left and selected notification details on the right
- Run the app on an iPhone.
- Go to the Notifications tab.
- Tap on any notification.
- [ ] Ensure the notification details are displayed fullscreen and there is no bottom tab bar visible

## Regression Notes
1. Potential unintended areas of impact
Notifications tab, Notification Details screen.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
UI changes only

PR submission checklist:


Testing checklist:
